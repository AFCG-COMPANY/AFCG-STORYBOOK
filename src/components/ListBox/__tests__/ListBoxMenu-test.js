import React from 'react';
import { mount } from 'enzyme';
import ListBox from '../';

describe('ListBoxMenu', () => {
  it('should render', () => {
    const wrapper = mount(
      <ListBox.Menu>
        <ListBox.MenuItem>Hello</ListBox.MenuItem>
      </ListBox.Menu>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
