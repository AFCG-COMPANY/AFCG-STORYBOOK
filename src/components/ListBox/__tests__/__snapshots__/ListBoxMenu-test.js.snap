// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListBoxMenu should render 1`] = `
<ListBoxMenu>
  <div
    className="bx--list-box__menu"
  >
    <ListBoxMenuItem
      isActive={false}
      isHighlighted={false}
    >
      <div
        className="bx--list-box__menu-item"
      >
        Hello
      </div>
    </ListBoxMenuItem>
  </div>
</ListBoxMenu>
`;
